dbms/src/Common/MemoryAllocTrace.cpp

```diff
@@ -16,14 +16,17 @@
 #include <common/config_common.h> // Included for `USE_JEMALLOC`
 
 #ifdef USE_JEMALLOC
+#if USE_JEMALLOC == 1
```

Comment:
I don't get the point. Why add "#if USE_JEMALLOC == 1" works here?

Reply:
Because it could be 0